projects/arlas-toolkit/src/lib/tools/color-generator-loader.ts

```diff
@@ -16,20 +16,24 @@ import * as tinycolor from 'tinycolor2';
  * If an external `externalColorsSaturationWeight` list is given to the service `getColor` method,
  * it is used insted of `colorsSaturationWeight`
  */
-@Injectable()
-export class ArlasColorGeneratorLoader implements ColorGeneratorLoader {
-  // TODO: keysToColors should be private and only accessed with setters and getters
```

Comment:
This could be interesting to keep or to apply to not lose that information

Reply:
this is no longer accurate, we need to access to keysToColors in donut code when we update the color